.ci/docker/common/install_cuda_aarch64.sh

```diff
@@ -3,19 +3,8 @@
 
 set -ex
 
-NCCL_VERSION=v2.21.5-1
-CUDNN_VERSION=9.5.1.17
-
-function install_cusparselt_062 {
-    # cuSparseLt license: https://docs.nvidia.com/cuda/cusparselt/license.html
-    mkdir tmp_cusparselt && pushd tmp_cusparselt
-    wget -q https://developer.download.nvidia.com/compute/cusparselt/redist/libcusparse_lt/linux-sbsa/libcusparse_lt-linux-sbsa-0.6.2.3-archive.tar.xz
-    tar xf libcusparse_lt-linux-sbsa-0.6.2.3-archive.tar.xz
-    cp -a libcusparse_lt-linux-sbsa-0.6.2.3-archive/include/* /usr/local/cuda/include/
-    cp -a libcusparse_lt-linux-sbsa-0.6.2.3-archive/lib/* /usr/local/cuda/lib64/
-    popd
-    rm -rf tmp_cusparselt
-}
+NCCL_VERSION=v2.26.2-1
+CUDNN_VERSION=9.8.0.87
```

Comment:
What about the common CUDNN install?

Reply:
This is only cuda 12.8 aarch64 bulds. We should really remove this script and relay on install_cuda.sh instead. Will look into thi

Reply:
@Skylion007 this is a cherry-pick, but I agree, we should have a common install script on trunk